live-build/build.gradle

```diff
@@ -48,46 +48,103 @@ def artifactTypes = ["aws": "vmdk",
                      "gcp": "gcp.tar.gz",
                      "kvm": "qcow2"]
 
+def configDir = "configuration"
+
+def upgradeArtifactsTaskSuffix = "UpgradeArtifacts"
+def upgradeArtifactsRunType = "upgrade-artifacts"
+def vmArtifactsTaskSuffix = "VmArtifacts"
+def vmArtifactsRunType = "vm-artifacts"
+def allTaskSuffix = ""
```

Comment:
The `*Suffix` variables seem to only be used in one spot each. Can we inline them?